qubes/tests/vm/dispvm.py

```diff
@@ -69,6 +70,15 @@ def setUp(self):
         self.app.domains[self.appvm.name] = self.appvm
         self.app.domains[self.appvm] = self.appvm
         self.addCleanup(self.cleanup_dispvm)
+        self.emitter = qubes.tests.TestEmitter()
+        # TODO: Ben: cleanup
+        # self.appvm_emitter = qubes.tests.TestEmitter()
+        # self.app.domains[self.appvm].fire_event = self.emitter.fire_event
```

Comment:
This should work for checking if events are fired, but won't fire actual handlers. But looking at the unit test results, it seems you've got actual event handlers working too:
```
qubes.tests.vm.dispvm/TC_00_DispVM/test_000_from_appvm_preload_only ... ok
started
ERROR (QubesValueError: QubesValueError("Invalid preload-dispvm value: non qube(s): 'disp42'"))
ERROR (QubesValueError: Invalid preload-dispvm value: non qube(s): 'disp42')
qubes.tests.vm.dispvm/TC_00_DispVM/test_000_from_appvm_preload_reject_max ... started
ok
```